### PR TITLE
[13.0][MIG] account_bank_statement_import_txt_xlsx

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by JoanSForgeFlow — APPROVED** [review]
Tested on local, LGTM :+1:

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow I feel like improving this module heavily this week for v12, jFYI

**Comment by MiquelRForgeFlow** [comment]
> for v12, jFYI

@alexey-pelykh why v12? And what kind of changes? Proceed carefully, because we already have this module in production in v12

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow hm, really heavy ones, but I guess there will be a migration. Why for v12 - I don't think we're going to use v13. Alternatively, I can then port the changes to v13 separately with migration script. I'd say that "total refactor" type of changes

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow IMHO, if you're using this in limited number of production instances, to wait for my PR and I'll provide a migration script 100%

**Comment by alexey-pelykh** [comment]
It would be great to get #264 in

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow if possible, please include recent changes to v12 into the PR

**Comment by MiquelRForgeFlow** [comment]
Depends of https://github.com/OCA/web/pull/1573.

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow `AssertionError: Element odoo has extra content: record, line 11`

**Comment by MiquelRForgeFlow** [comment]
@alexey-pelykh @JordiBForgeFlow ready to review and merge :)

File: account_bank_statement_import_txt_xlsx/models/account_bank_statement_import.py [hunk 1]
```diff
@@ -24,13 +24,12 @@ def _get_default_mapping_id(self):
...
 
-    @api.multi
     def _parse_file(self, data_file):
         self.ensure_one()
         try:
             Parser = self.env["account.bank.statement.import.sheet.parser"]
-            return Parser.parse(self.sheet_mapping_id, data_file, self.filename)
-        except:
+            return Parser.parse(data_file, self.sheet_mapping_id)
+        except Exception:
```
Comment (alexey-pelykh) [inline account_bank_statement_import_txt_xlsx/models/account_bank_statement_import.py]: `except BaseException:` would be equivalent of `except:`

Comment (MiquelRForgeFlow) [reply account_bank_statement_import_txt_xlsx/models/account_bank_statement_import.py]: done

**Review by alexey-pelykh — APPROVED** [review]
Code LGTM

**Comment by alexey-pelykh** [comment]
@MiquelRForgeFlow there's linter issue:
```
-    def _parse_rows(  # noqa: C901
-        self, mapping, currency_code, csv_or_xlsx, columns
-    ):
+    def _parse_rows(self, mapping, currency_code, csv_or_xlsx, columns):  # noqa: C901
```

**Review by LoisRForgeFlow — APPROVED** [review]
Tested locally, works fine :+1:

**Pull request merged**